modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss

```diff
@@ -44,21 +55,20 @@
 	}
 }
 
-@media (max-width: 768px) {
-	.report-cards-shortcut {
-		display: none;
+#wrapper .portlet-layout .portlet-body {
+	.lfr-ddm__form-report__header,
+	.lfr-ddm__form-report__tabs {
+		.container-fluid-max-xl {
+			max-width: 808px;
+		}
 	}
-}
-
-@media (min-width: 768px) {
-	.lfr-de__form-report {
-		display: flex;
+	.lfr-ddm__form-report__cards {
```

Comment:
This class should be declared in the `modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/Report.scss`, since it is not being used into the `FormReport` component